DRAFT-website-privacy-disclaimer.md

```diff
@@ -0,0 +1,32 @@
+# Linux Australia Website Privacy Policy and Disclaimer 
+
+## What does this Privacy Policy and Disclaimer apply to? 
+
+This Policy applies to the Linux Australia, Inc. website 
+
+_FOR DISCUSSION: Should this policy also apply to linux.conf.au? What about other conference websites? Do they have their own privacy policies? Should this be generic enough to be able to be used by other conferences / events? It might be a useful "value add" that we can provide. That begs the additional question of whether there should be a requirement on conferences and events to have a Privacy Policy, just like we require a link on their website back to the Linux Australia website?_
+
+## What type of personal information do we collect and store? 
+
+As a visitor to this website, we track the following information: 
+
+* Your IP address
+* The time and date at which you visited the website 
+* The type of browser ("user agent") you used to visit the website
+
+In addition, if you fill out forms on this website, we track the following information: 
+
+* Your email address
+* If provided, your contact information such as phone number and address
+* Details such as your name, title and affiliations
+* The information you submit in the form 
+
+In addition, if you submit a membership application, we track the following information: 
+
+* Your provided phone number and address 
+
+In addition, if you are accepted as a member, we track the following information:
+
+
+Your information is stored in a secure database, which is hosted in Australia. 
+
```

Comment:
Is this space meant to be blank?

Reply:
@lloy0076 This is a draft PR, it's not ready for feedback and review yet.

Reply:
Oh 😸